app/Http/Controllers/PaginaBeheerController.php

```diff
@@ -95,6 +95,8 @@ private function validateData(Request $reqeust){
             'EN_text' => 'required',
             'content_nl' => 'required',
             'content_en' => 'required',
+            'parentItem' => 'required',
+            'afterItem' => 'required',
```

Comment:
Bij het menu kan er ook gekozen worden om een menu item niet ergens na te plaatsen zo deze kan ook niet verplicht zijn.

Daarnaast in het dropdown menu is het eerst item ook niet na iets

Reply:
De repository gaat er altijd vanuit dat `afterItem` bestaat. Indien `itemType` gelijk is aan `subItem`, moet de `parentItem` ook defined zijn. Ik heb dit aangepast in mijn laatste commit. Ook heb ik nog een test case toegevoegd om alle edge cases hiervan te testen.